iocore/net/Makefile.am

```diff
@@ -136,9 +136,11 @@ test_libinknet_LDADD += \
 endif
 
 libinknet_a_SOURCES = \
-	AcceptOptions.cc \
-	AcceptOptions.h \
+  AcceptOptions.cc \
```

Comment:
I feel like I saw this indent width difference on another PR.

Reply:
Auto formatting fail.  I removed these whitespace changes.